workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Dashboard Basics
+---
+
+The Dashboard Builder allows you to generate new dashboards in under a minute, integrate custom widgets into existing dashboards, or effortlessly clone default dashboards for further refinement. Drill into any data ported to Middleware with advanced filtering options by date range and metric value.
```

Comment:
Recommended edit for the last sentence: "Drill into any data ported ingested into Middleware with advanced filtering options by date range, metric value, or any metric attribute."